### PR TITLE
Add an example demonstrating a one-off task

### PR DESCRIPTION
**Pull request by mheadd**
## Changes proposed in this pull request:
- Add a new example app that demonstrates how to deploy a one-off task

## security considerations
- None

### CODE REVIEW

File: cron-task/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Running one-off tasks on cloud.gov
+
+This is an example of a one-off task that can be run on cloud.gov. More on how to do this can be [found in the docs](https://cloud.gov/docs/management/one-off-tasks/). Note, this example uses node.js, but you can use any language that has a cron-like library to do something similar.
+
+## Usage
+
+To run this app locally, use `npm start`. To deploy to cloud.gov, use the instructions below:
+
+```bash
+~$ cf push -f manifest.yml --health-check-type none --no-route
```
Comment (apburnes) [inline cron-task/README.md:10]: The `health check` and `no route` can also be added in the manifest as well.  Not sure if this is worth pointing out.

File: cron-task/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+```
+
+When done, wait a few minutes for some log messages to show up, then check the logs:
```
Comment (apburnes) [inline cron-task/README.md:13]: Maybe we add `cf tasks task-runner` to view all the tasks ran. 🤷

File: cron-task/README.md (hunk 1 above)
Comment (mheadd) [reply cron-task/README.md:10]: Ah, that's. good idea.

File: cron-task/README.md (hunk 2 above)
Comment (mheadd) [reply cron-task/README.md:13]: Yes, adding this.

**Pull request merged**